src/workflow-engine/charms/temporal-worker/oci_factory/notification/mattermost_notifier.py

```diff
@@ -0,0 +1,93 @@
+import json
+import logging
+import os
+import requests
+import sys
+from typing import Optional
+
+
+logging.basicConfig(level=os.environ.get("TWC_LOG_LEVEL", "info").upper())
+
+COLORS = {
+    "success": "#33CC33",
+    "failure": "#CC0000",
+    "unknown": "#B0B0B0",
+}
+
+REQUIRED_ENVS = ["MATTERMOST_CHANNEL_ID", "MATTERMOST_TOKEN", "MATTERMOST_SERVER"]
+
+missing_envs = [env for env in REQUIRED_ENVS if os.environ.get(env) is None]
```

Comment:
why don't use any function in python?

Reply:
https://www.programiz.com/python-programming/methods/built-in/any

Reply:
with a lambda

Reply:
Using the list comprehension here makes it possible to print an error message informing the user all the missing environmental variables. I don't see a possibility using `any` could achieve so without over-complicating the code.